### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by Ecdldaiiere** [aclap-dev/video-downloadhelper#1541] [License & Purchase requests]
### What's your issue?

Hello, I just bought the $18 lifetime license with the intention of using it to convert a flv stream from a TikTok Live to an MP4. After I bought it however, it failed to do the task at hand. There was an error message saying it could not be done. I would like a refund.

I have also sent an email to the licenses@downloadhelper.net email but am posting here as well just to be safe.

### The 10 first digits (only 10) of your license number or email address

f4fe05ecf5

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, I replied to your email.

**Reply by Ecdldaiiere** [reply]
Thank you very much for the timely response. I'm sorry about the trouble.
